source/components/atoms/Input/Input.tsx

```diff
@@ -17,10 +17,7 @@ type InputProps = Omit<TextInputProps, 'onBlur'> & {
 const StyledTextInput = styled.TextInput<InputProps>`
   width: 100%;
   font-weight: ${({ theme }) => theme.fontWeights[0]}
-  background-color: ${props =>
-    props.colorSchema === 'neutral'
-      ? props.theme.colors.neutrals[5]
-      : props.theme.colors.complementary[props.colorSchema][2]};
+  background-color: ${props => props.theme.colors.complementary[props.colorSchema][2]};
```

Comment:
What does [2] mean?

Reply:
In the theme, the theme.colors.complementary has one array of color values per color, so we are just selecting the third value from that array.

Reply:
Hmm. I wish this was a bit more verbose in the code. Now I've to think...